ms_active_directory/core/ad_session.py

```diff
@@ -1686,10 +1689,18 @@ def find_groups_for_entities(self, entities: List[Union[str, ADObject]], attribu
                                                                              skip_validation)
 
         filter_pieces = []
-        for entity_dn in entity_dns_to_entities:
+        mapping_dict = {}
+        for entity_dn, entity in entity_dns_to_entities:
```

Comment:
this needs to be `entity_dns_to_entities.items()`

Reply:
Done.